### ISSUE TITLE
Issue with missing reconnection VODs due to stream disconnect

### ISSUE THREAD

**Issue by knakamura8** [Shishkebaboo/VodRecovery#19]
I have noticed that some VODs terminate upon receiving a disconnect (NotLikeThis) screen.  For example, on a twitchtracker stream, it will say that there is a single stream on a day for 3 hours (example).  However, let's assume that the streamer had a momentary disconnection 1 hour into the stream.  The m3u8 file will state that the twitch-total-secs will be 60 * 60 == 3600, and the ts chunks will only be relevant to that initial hour.  The m3u8 file should rather have 3 (hours) * 60 * 60 = 10800 as the total seconds.  Understandably, the last few ts chunks of a stream will include a NotLikeThis blinking screen, and that is it.  Are separate top-level m3u8 files generated when streams reconnect?

In summary, let's image user bobstreamer with twitchtracker stream such as https://twitchtracker.com/bobstreamer/streams/123456

The m3u8 will be VodRecovery_bobstreamer_123456.m3u8

But this file will only contain ts segments and length up to the first disconnection.  Are there extra m3u8s that are generated, such as VodRecovery_bobstreamer_123456_1.m3u8 ?  I am not familiar with how these m3u8s are generated/stored.

**Comment by knakamura8** [comment]
Nevermind, the issue appears to be with both twitchtracker and scullygnome.  For some reason, they do not pick up on stream restarts.  streamscharts correctly detects reconnections.

**Comment by Shishkebaboo** [comment]
@knakamura8 

Just a note on this issue... I am working on some functionality that will let you know when a stream isnt the full thing and will provide you the link to streamscharts.

**Comment by knakamura8** [comment]
@Shishkebaboo Thanks, that sounds very useful!  Actually I noticed that something seems strange with streamcharts.  Even though streamcharts detects multiple videos, something is not properly processing with the timestamps.  Error such as strptime() argument 1 must be str, not None.  Do not have the time to look into it at the moment, but perhaps in a day or two I will be able to notice, if you have not seen this yet.  Will reopen anyway.  Thanks for your hard work!

**Comment by Shishkebaboo** [comment]
@knakamura8 

I am thinking your running into the same thing as the following:  https://github.com/Shishkebaboo/VodRecovery/issues/18

**Comment by knakamura8** [comment]
Ha! I just saw that (had the tab open but forgot).  Thank you!
